storage/innobase/fil/fil0fil.cc

```diff
@@ -10673,8 +10626,8 @@ byte *fil_tablespace_redo_create(byte *ptr, const byte *end,
 
 #else  /* !UNIV_HOTBACKUP */
 
-<<<<<<< HEAD
-  const auto files = fil_system->get_scanned_files(page_id.space());
+  const auto files =
+      fil_system->get_scanned_filename_by_space_id(page_id.space());
```

Comment:
Why we changed from `get_scanned_files` to `get_scanned_filename_by_space_id` ?

Reply:
ms 8.0.22 change get_scanned_files to get_scanned_filename_by_space_id